Makefile

```diff
@@ -951,12 +888,12 @@ CMAKE_SHAREDMODS := sharedmods ext_pyston
 
 .PHONY: pyston_dbg pyston_release
 pyston_dbg: $(CMAKE_SETUP_DBG)
-	$(NINJA) -C $(CMAKE_DIR_DBG) pyston copy_stdlib copy_libpyston $(CMAKE_SHAREDMODS) ext_cpython $(NINJAFLAGS)
+	$(NINJA) -C $(CMAKE_DIR_DBG) pyston copy_stdlib copy_libpyston $(CMAKE_SHAREDMODS) ext_cpython clang-format unittests $(NINJAFLAGS)
```

Comment:
Hmm this seems like the wrong place to place the dependency.  I assume it's for the check/quick_check targets?  Maybe we should have those build `check-deps` rather than getting it through the pyston_dbg dependency.  (ex I also wonder if `make check` would have the issue that these dependencies aren't listed for the release build.)